cmd/root.go

```diff
@@ -41,7 +43,7 @@ func Execute() {
 
 func init() {
 	rootCmd.SilenceUsage = true
-	rootCmd.PersistentFlags().BoolVarP(&debug, "debug", "d", false, "enable debug mode")
+	rootCmd.PersistentFlags().CountVarP(&debugCount, "debug", "d", "enable debug mode")
```

Comment:
What do you say if we change CountVarP to an integer value which will correspond to debug level

Personally I am not a fan of `-xxxx` syntax, to me `-d 4` makes more sense and it is consistent with the long/short flags notation `--debug/-d`

Reply:
All this is provided by cobra/pFlags - [here](https://github.com/spf13/pflag/blob/master/count_test.go#L20)

supported today to give 2 as example: `-dd` / `-d -d` / `-d=2` / `--debug -d` / `--debug --debug` / `--debug=2`

Not working `-d 2` or `--debug 2` .. but believe this is a change that needs to happen to pFlag/Cobra upstream

The alternative is to make it an int, but that breaks the existing debug flag syntax

Reply:
ah, I didn't know that `-d=2` works. then its fine